src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -449,7 +450,7 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
               <Spacer mt={1} />
             )}
 
-          {avalaraPhase2Enabled && (
+          {avalaraPhase2Enabled && !isSold && (
```

Comment:
Maybe we can reuse [`artworkEcommerceAvailable`](https://github.com/artsy/force/blob/55baef7ada76d6a3072a1bfd242d644869d5f649/src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx#L400-L404) which takes into account [only for sale](https://github.com/artsy/gravity/blob/1cd2ea38e35729e38fe98b34bbc84363d047b872/app/models/util/for_sale.rb#L285-L300) artworks.

Reply:
Yes, this makes sense. Would be weird to have the tax message on artworks that aren't even for sale. Thanks!